package.json

```
@@ -12,6 +12,9 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "lint": "tslint -p .",
+    "generate:proxy-models": "npm-run-all generate:proxy:*",
```

```suggestion
    "generate:proxy-models": "npm-run-all generate:proxy:*",
```

proxy was the old name of io-backend, use 'backend' instead